Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -810,6 +810,19 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+    public override func visit(_ node: PatternBindingSyntax) -> SyntaxVisitorContinueKind {
+        if shouldSkip(node) {
+            return .skipChildren
+        }
+
+        if let unexpected = (node.pattern.as(TuplePatternSyntax.self)?.unexpectedBetweenElementsAndRightParen),
+            let previousToken = node.previousToken(viewMode: .sourceAccurate) {
+            addDiagnostic(unexpected, SpaceSeparatedIdentifiersError(firstToken: previousToken, additionalTokens: []))
```

Comment:
This will not use `constant declaration` but variable. 
We could add `nameForDiagnostics` where it's possible to add a string a string. 

And we could also add `helperText` where it's we can add `"is there an accidental break?"`

Or should we add `is there an accidental break?` for all `SpaceSeparatedIdentifiersError`